app/models/pdc_serialization/datacite.rb

```diff
@@ -215,11 +215,11 @@ def rights_from_work_resource(resource)
           rights
         end
 
-        def funding_reference_from_work_resource(resource)
-          return nil unless resource.award_number.present? && resource.funder_name.present?
-          award = ::Datacite::Mapping::AwardNumber.new(uri: resource.award_uri, value: resource.award_number)
-          funding_reference = ::Datacite::Mapping::FundingReference.new(name: resource.funder_name, award_number: award)
-          [funding_reference]
+        def funding_references_from_work_resource(resource)
+          resource.funders.map do |funder|
+            award = ::Datacite::Mapping::AwardNumber.new(uri: funder.award_uri, value: funder.award_number)
+            ::Datacite::Mapping::FundingReference.new(name: resource.funder_name, award_number: award)
```

Comment:
Shouldn't this be funder.funder name here?  I feel like this code must be untested.

Reply:
thanks! will do.